tests/PySys/environments/cumulocity.py

```diff
@@ -0,0 +1,225 @@
+import base64
```

Comment:
We already have c8y environment right, why do we need this?

Reply:
It makes sense to move this class to a separate file to separate concerns. It is also unrelated to the tede-environment. Splitting this into a module (and later on maybe a package) it will make it easier to introduce the python c8y api.